.github/workflows/build_all.yml

```diff
@@ -54,6 +54,29 @@ jobs:
         run: |
           python tools/sanity_checks.py
 
+  VisualStudio-clang-cl:
+    if: github.event_name != 'schedule' || github.repository == 'mesonbuild/wrapdb'
+    runs-on: windows-latest
+    env:
+      CC: clang-cl
+      CXX: clang-cl
```

Comment:
Probably we can do this DRY with a matrix, since the only difference compared to the VisualStudio job is CC/CXX, right?

Reply:
that's the only difference, yes.

Not sure what you mean with DRY.

Reply:
DRY -- "Don't Repeat Yourself"